generate-stackbrew-library.sh

```diff
@@ -9,37 +9,47 @@ array_6_6='6 latest';
 
 cd $(cd ${0%/*} && pwd -P);
 
+self="$(basename "$BASH_SOURCE")"
+
 versions=( */ )
 versions=( "${versions[@]%/}" )
-url='git://github.com/nodejs/docker-node'
+url='https://github.com/nodejs/docker-node'
+
+# sort version numbers with highest first
+IFS=$'\n'; versions=( $(echo "${versions[*]}" | sort -r) ); unset IFS
+
+# get the most recent commit which modified any of "$@"
+fileCommit() {
+	git log -1 --format='format:%H' HEAD -- "$@"
+}
 
-echo '# maintainer: Node.js Docker Team <https://github.com/nodejs/docker-node> (@nodejs)'
+echo "# this file is generated via ${url}/blob/$(fileCommit "$self")/$self"
+echo
+echo "Maintainers: The Node.js Docker Team <${url}> (@nodejs)"
+echo "GitRepo: ${url}.git"
+echo
 
 for version in "${versions[@]}"; do
 	if [[ "$version" == "docs" ]]; then
 		continue
 	fi
 	eval stub=$(echo "$version" | awk -F. '{ print "$array_" $1 "_" $2 }');
-	commit="$(git log -1 --format='format:%H' -- "$version")"
+	commit="$(fileCommit "$version")"
 	fullVersion="$(grep -m1 'ENV NODE_VERSION ' "$version/Dockerfile" | cut -d' ' -f3)"
 
 	versionAliases=( $fullVersion $version ${stub} )
-
+	echo "Tags: ${versionAliases[@]}"
```

Comment:
These are outputting space-separated but need to be comma-separated (hence why most of our scripts have that simple `join()` function).

ie, you've got `Tags: 6.6.0 6.6 6 latest` vs `Tags: 6.6.0, 6.6, 6, latest`